src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -1340,23 +1335,29 @@ public void testCutAndPasteNode() throws Exception {
 
         // add modification on node "node1"
         String createTwoWindingsTransformerAttributes = "{\"type\":\"" + ModificationType.TWO_WINDINGS_TRANSFORMER_CREATION + "\",\"equipmentId\":\"2wtId\",\"equipmentName\":\"2wtName\",\"seriesResistance\":\"10\",\"seriesReactance\":\"10\",\"magnetizingConductance\":\"100\",\"magnetizingSusceptance\":\"100\",\"ratedVoltage1\":\"480\",\"ratedVoltage2\":\"380\",\"voltageLevelId1\":\"CHOO5P6\",\"busOrBusbarSectionId1\":\"CHOO5P6_1\",\"voltageLevelId2\":\"CHOO5P6\",\"busOrBusbarSectionId2\":\"CHOO5P6_1\"}";
+        String networkModificationResultJson = mapper.writeValueAsString(Optional.of(NetworkModificationResult.builder()
```

Comment:
The node is not built so you shouldn't return a NetworkModificationResult

Reply:
Done